docs-src/references/sdk-metrics.md

```diff
@@ -7,10 +7,12 @@ tags:
   - reference
 ---
 
-> For Cluster metrics, see [Cluster ▶️ Production deployment ▶️ Scaling and Metrics](/server/production-deployment/#scaling-and-metrics).
-> For Cloud metrics, see [Temporal Cloud ▶️ Cloud metrics](/cloud/how-to-monitor-temporal-cloud-metrics).
+> See also: [References ▶️ Cluster metrics](/references/cluster-metrics).
 
 The Temporal SDKs emit a set of metrics from Temporal Client usage and Worker Processes.
+To set up metrics, see [Dev guide ▶️ Observability ▶️ Metrics](/application-development/observability#metrics).
```

Comment:
```suggestion
To set up metrics, see the [Metrics section of the Dev guide](/app-dev-context/metrics).
```

That section is built by Assembly so we can do this above and it will rewrite with a link preview